docs/feature_split_keyboard.md

```diff
@@ -386,6 +386,15 @@ This sets how many LEDs are directly connected to each controller.  The first nu
 
 Enabling this option changes the startup behavior to listen for an active USB communication to delegate which part is master and which is slave. With this option enabled and theres's USB communication, then that half assumes it is the master, otherwise it assumes it is the slave.
 
+```c
+#define SPLIT_USB_FORCE_LEFT
+#define SPLIT_USB_FORCE_RIGHT
+```
+
+Forces the usb connection to be enabled on the corresponding half. If both are set, left wins over right.
+
+?> Only set these options if `SPLIT_USB_DETECT` does not work as intended.
+
 Without this option, the master is the half that can detect voltage on the physical USB connection (VBUS detection).
 
 Enabled by default on ChibiOS/ARM.
```

Comment:
The description for the new options is inserted in a wrong position — e.g., the “Enabled by default on ChibiOS/ARM” paragraph is a part of the `SPLIT_USB_DETECT` description. This should probably be inserted after the `SPLIT_USB_TIMEOUT_POLL` section (so that the documentation for parameters that modify the `SPLIT_USB_DETECT` behavior is kept together with the description of `SPLIT_USB_DETECT`).